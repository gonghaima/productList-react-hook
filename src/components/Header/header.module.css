@import "../../config.css";

header h2 {
  margin: 0;
  color: var(--h2-header-color);
}
header summary {
  color: var(--header-summary-color);
  text-align: center;
  line-height: 28px;
}
header select {
  height: 28px;
  color: var(--header-select-color);
  border: 0;
}
.displaySelection {
  display: flex;
  justify-content: space-between;
}
