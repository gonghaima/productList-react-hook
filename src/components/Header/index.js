import React from "react";
import headerStyles from "./header.module.css";
import Selection from "../Selection";

export default props => {
  return (
    <header>
      <h2>All Products</h2>
      <div className={headerStyles.displaySelection}>
        <summary>1000 products</summary>
        <Selection />
      </div>
    </header>
  );
};
