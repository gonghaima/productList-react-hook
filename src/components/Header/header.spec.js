import React from "react";
import { shallow } from "enzyme";
import Header from "./index";

it("Header component renders without crashing", () => {
  const HeaderComponent = shallow(<Header />);
  expect(HeaderComponent.debug()).toMatchSnapshot();
});
