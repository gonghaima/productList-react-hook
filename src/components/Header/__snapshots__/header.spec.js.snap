// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders without crashing 1`] = `
"<header>
  <h2>
    All Products
  </h2>
  <div className=\\"displaySelection\\">
    <summary>
      1000 products
    </summary>
    <withRouter(Selection) />
  </div>
</header>"
`;
